aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/BaseHandlerStd.java

```diff
@@ -136,6 +136,26 @@ protected ProgressEvent<ResourceModel, CallbackContext> applyParameters(final Am
                 .then(progressEvent -> modifyParameters(progressEvent, currentDBParameters, proxy, proxyClient, requestLogger));
     }
 
+
+    protected ProgressEvent<ResourceModel, CallbackContext> applyParameters(final AmazonWebServicesClientProxy proxy,
+                                                                            final ProxyClient<RdsClient> proxyClient,
+                                                                            final ProgressEvent<ResourceModel, CallbackContext> progress,
+                                                                            final RequestLogger requestLogger) {
+        ResourceModel model = progress.getResourceModel();
+        CallbackContext callbackContext = progress.getCallbackContext();
+        //isParametersApplied flag for unit testing
+        if (callbackContext.isParametersApplied())
+            return ProgressEvent.defaultInProgressHandler(callbackContext, NO_CALLBACK_DELAY, model);
+        callbackContext.setParametersApplied(true);
+
+        Map<String, Parameter> defaultEngineParameters = Maps.newHashMap();
```

Comment:
NIT: let's leave a comment saying it would be populated in the upcoming call

Reply:
Done.